cases/management/commands/enrich_ciaa_allegations.py

```
@@ -0,0 +1,583 @@
+            )
+            return
+
+        try:
+            allegations = self._extract_allegations(
+                press_release_text=press_release_text,
+                case_title=case.title,
+                llm_model=llm_model,
+                llm_base_url=llm_base_url,
+                llm_api_key=llm_api_key,
+            )
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)